lib/torrent/scheduler/connstate/state.go

```diff
@@ -32,11 +32,27 @@ var (
 	ErrConnClosed              = errors.New("conn is closed")
 	ErrInvalidActiveTransition = errors.New("conn must be pending to transition to active")
 	ErrTooManyMutualConns      = errors.New("conn has too many mutual connections")
+
+	// This should NEVER happen.
+	errUnknownStatus = errors.New("invariant violation: unknown status")
 )
 
+type status int
+
+const (
+	_uninit status = iota
```

Comment:
can you add some comments about what each means

Reply:
I think pending / active are fairly obvious in the context of this file... There's a big comment above `State` which describes the semantics of pending / active.